mitosheet/src/mito/utils/actions.tsx

```diff
@@ -1483,9 +1483,21 @@ export const createActions = (
             searchTerms: ['import', 'custom import', 'user defined import'],
             tooltip: "Import data using the custom defined imports function "
         },
+        [ActionEnum.REPLACE]: {
+            type: ActionEnum.REPLACE,
+            shortTitle: 'Replace',
+            longTitle: 'Replace',
+            actionFunction: () => {
+                // TODO
```

Comment:
I think you can remove this action, as you already have a search one. You'll have to delete it from the ActionEnum